src/config.ts

```diff
@@ -0,0 +1,22 @@
+export const uiConfiguation = {
```

Comment:
I don't think that we need headlines for every group of variables (like uri, roles, etc.)

Reply:
It should be like any other global variables

Reply:
I don't agree to be honest.
sub-categories allow for better management and visibility going forward.